### PR TITLE
rename "port" to DHTport in config and hcdev flags, closes #640

### LINKED ISSUE #640: Change name of Port in config
People get confused about the difference between inter-node port and the UI port. 
- [ ] Change the name in config.json to DHTPort
- [ ] change name of flag in `hcdev` to match

### PR DESCRIPTION
**Pull request by zippy**

### CODE REVIEW

**Comment by zippy** [comment]
Hey @Connoropolous and @thedavidmeister added both of you to review on this one because it's another flag name change.  Art requested this one because "port" without qualification leads to confusion.  This will probably need docs updates.  Especially I think @philipbeadle you may have some guides and instructions that use the `-port` flag.

**Comment by thedavidmeister** [comment]
followup issue open: https://github.com/holochain/holochain-proto/issues/681

File: cmd/hcd/hcd.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ var verbose bool
 func setupApp() (app *cli.App) {
 	app = cli.NewApp()
 	app.Name = "hcd"
-	app.Usage = fmt.Sprintf("serve a chain to the web on localhost:<port> (defaults to %s)", defaultPort)
+	app.Usage = fmt.Sprintf("serve a chain to the web on localhost:<port> (defaults to %s)", defaultUIPort)
 	app.ArgsUsage = "holochain-name [port]"
```
Comment (thedavidmeister) [inline cmd/hcd/hcd.go]: should this say [ui-port], it shows up in help text?

File: cmd/hcd/hcd.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ var verbose bool
 func setupApp() (app *cli.App) {
 	app = cli.NewApp()
 	app.Name = "hcd"
-	app.Usage = fmt.Sprintf("serve a chain to the web on localhost:<port> (defaults to %s)", defaultPort)
+	app.Usage = fmt.Sprintf("serve a chain to the web on localhost:<port> (defaults to %s)", defaultUIPort)
```
Comment (thedavidmeister) [inline cmd/hcd/hcd.go]: `<ui-port>`?

File: cmd/hcd/hcd.go (hunk 1 above)
Comment (thedavidmeister) [reply cmd/hcd/hcd.go]: this also shows up in README.md:

> $ hcd <CHAIN_NAME> [PORT]

File: cmd/hcdev/hcdev.go [hunk 3]
```diff
@@ -644,7 +644,7 @@ func setupApp() (app *cli.App) {
 			Name:      "web",
 			Aliases:   []string{"serve", "w"},
 			ArgsUsage: "[port]",
-			Usage:     fmt.Sprintf("serve a chain to the web on localhost:<port> (defaults to %s)", defaultPort),
+			Usage:     fmt.Sprintf("serve a chain to the web on localhost:<port> (defaults to %s)", defaultUIPort),
```
Comment (thedavidmeister) [inline cmd/hcdev/hcdev.go]: `<ui-port>`?

File: cmd/hcdev/hcdev.go [hunk 4]
```diff
@@ -644,7 +644,7 @@ func setupApp() (app *cli.App) {
 			Name:      "web",
 			Aliases:   []string{"serve", "w"},
 			ArgsUsage: "[port]",
```
Comment (thedavidmeister) [inline cmd/hcdev/hcdev.go]: `[ui-port]`?

**Comment by zippy** [comment]
The DefaultPort for the bootstrap server (bs.go) seems just fine, that's a separate chunk of code.

**Pull request merged**